src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -311,4 +314,34 @@ public function theAdministratorShouldSeeThatThisOrderHasBeenPlacedIn(
 
         Assert::same($currencyCode, $currency);
     }
+
+    /**
+     * @Then I should see an order with :orderNumber number
+     */
+    public function iShouldSeeOrderWithNumber(string $orderNumber)
+    {
+        $order = $this->orderRepository->findOneBy(['number' => $orderNumber]);
+
+        Assert::notNull($order, sprintf('Cannot find order with number %s', $orderNumber));
```

Comment:
The check should be done on the response collection.
Something like [this](https://github.com/Sylius/Sylius/blob/1.13/src/Sylius/Behat/Context/Api/Admin/ManagingCatalogPromotionsContext.php#L1533).

Reply:
@NoResponseMate done